frontend/src/services/fetch/fetchers/searchFetcher.ts

```diff
@@ -4,19 +4,20 @@ import { fetchOpportunitySearch } from "src/services/fetch/fetchers/fetchers";
 import {
   PaginationOrderBy,
   PaginationRequestBody,
+  PaginationSortOrder,
   QueryParamData,
   SearchFetcherActionType,
   SearchFilterRequestBody,
   SearchRequestBody,
 } from "src/types/search/searchRequestTypes";
 
 const orderByFieldLookup = {
-  relevancy: "relevancy",
-  opportunityNumber: "opportunity_number",
-  opportunityTitle: "opportunity_title",
-  agency: "agency_code",
-  postedDate: "post_date",
-  closeDate: "close_date",
+  relevancy: ["relevancy"],
```

Comment:
Shouldn't this also have post_date? I see you had a default pagination sort order below, but if someone intentionally chose relevancy (not sure how that links / if that's possible) you'd lose the post_date bit, right?
```suggestion
  relevancy: ["relevancy", "post_date"],
```

Reply:
Yes, I'm torn on how to best handle the default and the post_date backup. Part of me thinks we should always have a post_date fallback, so even if sorting by agency, we show the ones closing sooner earlier amongst the peers from that same agency. But I think In the implementation you'll see that if it's hitting on relevancy it's adding the post_date backup in the pagination builder. But agree, I'm not sure if that's the right way, except to note, anything we default here, doesn't affect the "no sort" route which is possible and does happen (primarily if you've just never touched sort upon landing on the page).

Reply:
[tldr: I'm good with this change as is] it seems like based on the changes to the fetcher, the default is being set the way we want with the secondary sort. Beyond that, I feel like if users want to update the sort away from the default I think it's best to not implement a hidden secondary sort behind the scenes at the moment.